textproc/aha/Portfile

```
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                aha
```

The `github.setup` line implicitly sets both `name` and `version`.

```suggestion
github.setup        theZiz aha 0.5.1
```

...and then you can delete the `name` and `version` lines.

---

Thanks! I did this and pushed a new commit.